docs/reference/publishers/sendgrid.md

```diff
@@ -0,0 +1,14 @@
+# SendGrid
+A publisher that uses the Twilio SendGrid API to upload email templates to SendGrid.
+
+## Configuration
+You need to configure the following fields when you want to use the SendGrid publisher:
+
+* SendGrid API Key - Please refer to the [SendGrid docs](https://docs.sendgrid.com/ui/account-and-settings/api-keys) for how to create an API key. Make sure it has at least enough permission to create and update email templates.
+* Visualization endpoint - The visualization server transforms a composed email experience (JSON) into proper HTML that can be used in an email.
```

Comment:
Is this correct? And is our demo visualization endpoint something others can use so I can mention it here? 

We probably need some more background on what this does exactly. Maybe with an example of input and output. Thoughts?

Reply:
The visualization server needs to be hosted by the client because they need to modify it with their own templates / components.